pom.xml

```diff
@@ -16,14 +16,16 @@
 
 	<properties>
 		<spring-cloud-deployer.version>1.0.0.RC1</spring-cloud-deployer.version>
-		<spring-hadoop.version>2.4.0.M1</spring-hadoop.version>
+		<spring-cloud-stream.version>1.0.0.RC3</spring-cloud-stream.version>
```

Comment:
Ah, that is probably leftover when I had a local merge conflict.  I'll take it away.